src/AnalogInputFirmata.cpp

```diff
@@ -52,7 +52,8 @@ void AnalogInputFirmata::reportAnalog(byte analogPin, int value)
         // Send pin value immediately. This is helpful when connected via
         // ethernet, wi-fi or bluetooth so pin states can be known upon
         // reconnecting.
-        Firmata.sendAnalog(analogPin, analogRead(analogPin));
+        // Firmata.sendAnalog(analogPin, analogRead(analogPin));
+        Firmata.sendAnalog(analogPin, analogRead(analogInputToDigitalPin(analogPin)));
```

Comment:
This should really be fixed in the esp32 arduino library. analogRead should be able to accept the analog pin number or the digital pin number.